cmd/dep/godep_importer_test.go

```diff
@@ -364,10 +364,10 @@ func TestGodepConfig_ProjectExistsInLock(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		result := projectExistsInLock(lock, c.importPath)
+		result := projectExistsInLock(lock, gps.ProjectRoot(c.importPath))
 
 		if result != c.want {
-			t.Fatalf("projectExistsInLock result is not as expected: \n\t(GOT) %v \n\t(WNT) %v", result, c.want)
+			t.Errorf("projectExistsInLock result is not as expected: \n\t(GOT) %v \n\t(WNT) %v", result, c.want)
```

Comment:
Move this whole test function `TestGodepConfig_ProjectExistsInLock()` as part of `root_analyzer_test.go`?